Library/Formula/openshift-cli.rb

```diff
@@ -0,0 +1,36 @@
+class OpenshiftCli < Formula
+  desc "OpenShift command-line interface tools"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift/origin.git",
+    :tag => "v1.1.1",
+    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
+
+  head "https://github.com/openshift/origin.git"
+
+  depends_on "go" => :build
+
+  def install
+    # this is necessary to avoid having the version marked as dirty
+    ignore_file = buildpath/".git/info/exclude"
+    ignore_file.open("w") do |file|
+      file.puts "/.brew_home"
+    end
+
+    system "make", "all", "WHAT=cmd/openshift", "GOFLAGS=-v"
```

Comment:
The "oc" binary can be built directly and is smaller (~40M vs 100M).  However, to get oadm you effectively need "openshift" for now so this is probably better.  We plan to move oadm into oc at some point soon at which point it may make sense to switch this and we can circle back then.